roofit/roofit/inc/RooGaussExpTails.h

```diff
@@ -0,0 +1,48 @@
+#ifndef RooFit_RooFit_RooGaussExpTails_h
+#define RooFit_RooFit_RooGaussExpTails_h
+
+#include "RooAbsPdf.h"
+#include "RooRealProxy.h"
+
+class RooAbsReal;
+
+class RooGaussExpTails : public RooAbsPdf {
+public:
+  RooGaussExpTails() { };
+  RooGaussExpTails(const char *name, const char *title,
+                         RooAbsReal::Ref _x,
+                         RooAbsReal::Ref _x0,
+                         RooAbsReal::Ref _sigma,
+                         RooAbsReal::Ref _kL,
+                         RooAbsReal::Ref _kH
+                         );
+  RooGaussExpTails(const RooGaussExpTails& other, const char* name=nullptr);
+  TObject* clone(const char* newname) const override {
+    return new RooGaussExpTails(*this,newname);
+  }
+
+  Int_t getAnalyticalIntegral(RooArgSet& allVars, RooArgSet& analVars, const char* rangeName=nullptr) const override;
+  double analyticalIntegral(Int_t code, const char* rangeName=nullptr) const override;
+
+  RooAbsReal::Ref x() const { return *x_; }
+  RooAbsReal::Ref x0() const { return *x0_; }
+  RooAbsReal::Ref sigma() const { return *sigma_; }
+  RooAbsReal::Ref kL() const { return *kL_; }
+  RooAbsReal::Ref kH() const { return *kH_; }
+
+protected:
+  double evaluate() const override;
+
+private:
+  RooRealProxy x_;
+  RooRealProxy x0_;
+  RooRealProxy sigma_;
+  RooRealProxy kL_;
+  RooRealProxy kH_;
```

Comment:
```suggestion
  RooRealProxy _x;
  RooRealProxy _x0;
  RooRealProxy _sigma;
  RooRealProxy _kL;
  RooRealProxy _kH;
```
We should follow the convention of most RooFit classes to start the private members with an underscore